wom/src/main/scala/wdl4s/wdl/WdlCall.scala

```diff
@@ -55,25 +52,10 @@ object WdlCall {
 
   private def buildWomNodeAndInputs(wdlCall: WdlCall): ErrorOr[CallWithInputs] = {
 
-    def resolveVariable(v: AstTools.VariableReference): ErrorOr[(String, GraphNodePort.OutputPort)] = {
-      val name =  v.fullVariableReferenceString
-      for {
-        parent <- wdlCall.parent.toValidNel("Call has no parent?")
-        node <- parent.resolveVariable(v.terminal.sourceString).toValidNel(s"Failed to resolve variable $name")
-        outputPort <- outputPortFromNode(node, v.terminalSubIdentifier)
-      } yield name -> outputPort
-    }
-
-
     val graphNodeInputExpressionValidation: ErrorOr[List[GraphNodeInputExpression]] = {
-      val graphNodeInputExpression: ((String, WdlExpression)) => ErrorOr[GraphNodeInputExpression] = {
-        case (name, wdlExpression) =>
-          for {
-            resolvedVariables <- wdlExpression.variableReferences.toList traverse resolveVariable
-            womExpression = WdlWomExpression(wdlExpression, None)
-          } yield GraphNodeInputExpression(name, womExpression, resolvedVariables.toMap)
+      def graphNodeInputExpression: ((String, WdlExpression)) => ErrorOr[GraphNodeInputExpression] = {
+        case ((inputName, wdlExpression)) => WdlWomExpression.graphNodeInputExpression(inputName, WdlWomExpression(wdlExpression, None), wdlCall)
```

Comment:
They're needed because it's a single tuple argument rather than a tuple of single arguments.